src/machine.tex

```diff
@@ -88,6 +88,19 @@ \subsection{Machine ISA Register {\tt misa}}
 zero after two shifts, then the machine is RV64, else RV128.
 \end{commentary}
 
+\begin{commentary}
```

Comment:
Did you determine that a commentary block is sufficient for this?  i.e. do we not need normative text explaining MXLEN's effect on the PC?

Reply:
I think we need new normative text about the PC.  Working out the language with JohnH.